impeller/compiler/shader_lib/impeller/transform.glsl

```diff
@@ -0,0 +1,13 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#ifndef TRANSFORM_GLSL_
+#define TRANSFORM_GLSL_
+
+vec2 transform(mat4 matrix, vec2 point) {
```

Comment:
Please follow the naming convention present in `blending.glsl` for utilities with type variation. I'd recommend: `IPVec2TransformPosition` to distinguish from, say, `IPVec3TransformDirection`, which would be similar except with 0 for the homogeneous coordinate. Also, please add a doc string for all functions added to the shader library.

Reply:
Renaming is complete. 

Would you mind giving me a doc string?  Since I'm not a native speaker, I'm not sure how to describe it.

Reply:
Sure, something like this would be good:
```
/// Returns the Cartesian coordinates of `position` in `transform` space.
IPVec2TransformPosition(mat4 transform, vec2 position) {
```